.github/workflows/continuous-monitoring.yml

```diff
@@ -0,0 +1,79 @@
+name: Continuous Monitoring of Dockerhub and public ECR
+on:
+  schedule:
+    - cron:  '0 */6 * * *'
+
+jobs:
+  monitor-ecr:
+    name: Monitor public ECR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Pull an image from public ECR
+        run: docker pull public.ecr.aws/xray/aws-xray-daemon:latest
+
+      - name: Publish metric on pulling an image
+        if: ${{ always() }}
+        run: |
+          if [[ "$(docker images -q public.ecr.aws/xray/aws-xray-daemon:latest 2> /dev/null)" == "" ]]; then
```

Comment:
Oh one suggestion, it's probably better to use the status of the pull step rather than this check. A bit more direct, and will be easy to copy-paste into other projects.

Reply:
`outcome`

As an example I use it here

Reply:
Added!